vignettes/release-process.qmd

```diff
@@ -0,0 +1,312 @@
+---
+title: "Introduction to the release process"
+description: >
+  Reference for how to perform R package releases in the Hubverse
+vignette: >
+  %  \VignetteIndexEntry{The Release Process}
+  %  \VignetteEngine{quarto::html}
+  %  \VignetteEncoding{UTF-8}
+knitr:
+  opts_chunk: 
+    collapse: true
+    comment: '#>'
+---
+
+Parts of this document is adapted for The Hubverse from [The Carpentries Developer's Handbook](https://carpentries.github.io/workbench-dev/releases.html) (c) The Carpentries. 
```

Comment:
For each of the vignettes, maybe we should say explicitly that what license(s) the original documents (and the new hubverse ones) were released under. Looks like a CC one from the original at least.